DashWallet/Sources/UI/CrowdNode/Online/OnlineAccountDetailsController.swift

```diff
@@ -42,6 +42,8 @@ final class OnlineAccountDetailsController: UIViewController {
     }
 
     private func configureHierarchy() {
+        view.backgroundColor = .dw_secondaryBackground()
```

Comment:
This creates some issues with UI. By design, we have a white background there with gray address boxes: https://www.figma.com/file/84tUBQA9DYU1svyQx8tnbk/Dash-Wallet%E3%83%BBIn-process%E3%83%BBAndroid?node-id=10643%3A18770&t=z4LVy8U1iXqA1XMw-0

<img src="https://user-images.githubusercontent.com/7733989/218629177-09cd13c1-506b-44ab-ba7b-581f6a03d6d1.jpeg" width="300" />

Reply:
Fixed: Ended up introducing new color for blocks like 'Primary Dash address'